packages/functional-tests/tests/resetPassword/resetPasswordWithCode/resetPassword.spec.ts

```diff
@@ -0,0 +1,166 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { expect, test } from '../../../lib/fixtures/standard';
+
+// This test files contains the latest tests for reset password with code (React)
+// See FXA-9344
+
+test.describe('severity-1 #smoke', () => {
+  test.beforeEach(async ({ pages: { configPage } }) => {
+    const config = await configPage.getConfig();
+    test.skip(
+      config.featureFlags.resetPasswordWithCode !== true,
+      'see FXA-9728, remove conditional skip when feature flag removed'
+    );
+    test.slow();
```

Comment:
nit, does this need the `test.slow()` now that @Trinaa tweaked the global test timeout?

Reply:
Yep. Make sure to remove this!